src/DurableTask.SqlServer/Scripts/logic.sql

```diff
@@ -478,6 +478,20 @@ BEGIN
 END
 GO
 
+CREATE OR ALTER PROCEDURE dt.RewindInstance
+    @InstanceID varchar(100),
+    @Reason varchar(max) = NULL
+AS
+BEGIN
+    DECLARE @TaskHub varchar(50) = dt.CurrentTaskHub()
```

Comment:
Not used?

Reply:
Good catch, should have removed this line!